content/messaging/python-sdk/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+weight: 60
+category: "messaging"
+subcategory: "python-sdk"
+title: "LiveChat Python SDK"
+tagline: "Build external backend applications and extend LiveChat features."
+desc: "LiveChat Python SDK is a tool developers can use to build external backend applications."
+---
+
+# Introduction
+
+This Software Development Kit, written in Python, is a tool for developers to build external backend apps, which extend LiveChat features. The SDK makes it easy to use the [Agent Chat API](/messaging/agent-chat-api/), the [Customer Chat API](/messaging/customer-chat-api/), and the [Configuration API](/management/configuration-api/).
+
+# SDK & Docs
+
+The SDK is available on GitHub. The documentation is available in the _typedoc_ format. 
```

Comment:
```suggestion
The SDK is available on GitHub along with a usage example. Currently, the SDK documentation is available only in HTML, but soon it will be available in a more convenient format, _typedoc_.
```